src/client/sharer/index.coffee

```diff
@@ -108,32 +48,29 @@ class Sharer
     # 2. Find url from Click Targt
     # 3. Ship container node
 
-    opts.params.url ||= findUrl(event.target)
+    opts.params.url = opts.params.url || opts.params.href
+
+    if (!opts.params.url)
+      opts.params.url = findUrl(event.target)
+      opts.params.title = opts.params.title || domWalker.getMetaValue('og:title') || document.title
```

Comment:
When sharing an item from the homepage of a website `Hull.share({ provider: 'twitter': url: 'http://acme.com/item/123' })`, the text in the twitter sharer is the title of the homepage not the title of the item.

To me if `params.title` is `null` we should fetch the title of the provided url server side.